example/android/app/src/main/java/com/airbnb/android/react/navigation/example/MainFragment.java

```diff
@@ -38,7 +39,8 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
     btnScreen.setOnClickListener(new View.OnClickListener() {
       @Override
       public void onClick(View v) {
-        getScreenCoordinator().presentScreen(NativeFragment.newInstance(1));
+//        ReactNativeIntents.pushScreen(getActivity(), "ScreenOne");
+        getScreenCoordinator().pushScreen("ScreenOne");
```

Comment:
There are so. many. ways. to create these, and I'm not actually sure the right/best way. Alas, this works.